driver/pelican/main.go

```diff
@@ -210,6 +210,30 @@ func main() {
 				}
 			}
 		})
+
+		schedstatIfaces[i].SubscribeSlot("schedule", func(msg *bw2.SimpleMessage) {
```

Comment:
We'll also need to update the XBOS documentation to reflect the fact that a thermostat interface has a new slot.

Reply:
Ok! Should I make a pull request on the XBOS repo? I know I have one PR from before still in waiting, but I believe you and @gtfierro discussed that changes would be made with the next version of XBOS? Should I delay the update or just do that now?